Description: Initialize NSDate before playing with threads and locks.
Origin: http://lists.gnu.org/archive/html/gnustep-dev/2016-08/msg00002.html
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=833134
Author: Richard Frith-Macdonald

--- a/src/SDL/main.m
+++ b/src/SDL/main.m
@@ -25,6 +25,7 @@
 
 #ifdef GNUSTEP
 #import <Foundation/NSAutoreleasePool.h>
+#import <Foundation/NSDate.h>
 #import <Foundation/NSString.h>
 
 #import "GameController.h"
@@ -47,6 +48,10 @@
 {
 #ifdef GNUSTEP
 	int i;
+
+	// Initialize NSDate before playing with threads and locks.
+	// http://lists.gnu.org/archive/html/gnustep-dev/2016-08/msg00002.html
+	[NSDate class];
 	
 #if OOLITE_WINDOWS
 
