--- oolite-source-1.80/src/Core/Materials/OOPNGTextureLoader.m.orig	2015-02-27 20:42:50.061222004 +0000
+++ oolite-source-1.80/src/Core/Materials/OOPNGTextureLoader.m	2015-02-27 20:42:57.291221472 +0000
@@ -31,7 +31,6 @@
 #import "OOCPUInfo.h"
 #import "NSDataOOExtensions.h"
 
-void png_error(png_structp, png_const_charp) NO_RETURN_FUNC;
 
 
 static void PNGError(png_structp png, png_const_charp message);
