diff -up oolite-source-1.90/src/Core/OOCPUInfo.m.omv~ oolite-source-1.90/src/Core/OOCPUInfo.m
--- oolite-source-1.90/src/Core/OOCPUInfo.m.omv~	2025-03-09 15:31:16.660234067 +0000
+++ oolite-source-1.90/src/Core/OOCPUInfo.m	2025-03-09 15:32:33.090477146 +0000
@@ -116,6 +116,7 @@ Taken straight out of the x64 gcc's __cp
 */
 inline OO_GNU_INLINE void OOCPUID(int CPUInfo[4], int InfoType)
 {
+#if defined(__x86_64__) || defined(__i386__)
 	__asm__ __volatile__ (
 /* Fixes building on 32-bit systems where %EBX is used for the GOT pointer */
 #if (OOLITE_LINUX && !defined __LP64__)
@@ -129,6 +130,9 @@ inline OO_GNU_INLINE void OOCPUID(int CP
           : "=a" (CPUInfo [0]), "=b" (CPUInfo [1]), "=c" (CPUInfo [2]), "=d" (CPUInfo [3])
 #endif
           : "a" (InfoType));
+#else
+	memset(CPUInfo, 0, sizeof(int)*4);
+#endif
 }
 
 
